app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,18 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
+    <title><%= full_title(yield(:title)) %></title>
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all',
```

Comment:
- Dùng " thay cho ' trong toàn bộ file này
- Sau stylesheet_link_tag cách 1 space thôi em